src/fetch/period_slippage.py

```diff
@@ -134,6 +105,7 @@ def get_period_slippage(
     Executes & Fetches results of slippage query per solver for specified accounting period.
     Returns a class representation of the results as two lists (positive & negative).
     """
+    update_token_list(dune, fetch_trusted_tokens())
```

Comment:
What happens if this is not successful? Will the script stop under all circumstances? E.g. if dune refusing creating a table for some reaon, how would the user be warned?

Reply:
Maybe you can describe this in a comment

Reply:
Maybe you can describe this in a comment

Reply:
Yea. This would raise an error from the dune API. There is a check if "errors" exist in the return data. I could put an assertion in the update method as well that the size of the return data is equal to the size of the token list passed into it.